core/templates/domain/review_test/review-test.model.spec.ts

```diff
@@ -13,26 +13,20 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for Review test object factory.
+ * @fileoverview Unit tests for ReviewTest.
  */
 
-import { ReviewTestObjectFactory } from
-  'domain/review_test/review-test-object.factory';
+import { ReviewTest } from
+  'domain/review_test/review-test.model';
```

Comment:
```suggestion
import { ReviewTest } from 'domain/review_test/review-test.model';
```
Ditto for all other places where we can simplify this*

Reply:
done here and elsewhere!